packages/clerk-js/src/ui/utils/mergeRefs.ts

```diff
@@ -0,0 +1,10 @@
+export const mergeRefs = <T>(...refs: React.RefObject<T>[]) => {
```

Comment:
❓ Can you please elaborate on this?

Reply:
This is used in case we want to forward a ref in the component, and also `useRef` inside the component itself for some other reason.